### PR TITLE
Logging for monitoring logs

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds some logging so that these logs can be monitored via Splunk reports

### CODE REVIEW

**Review by irfanuddinahmad — APPROVED** [review]
Looks Good! ... once build is green

File: enterprise_data/clients.py [hunk 1]
```diff
@@ -45,15 +45,28 @@ def get_enterprise_learner(self, user):
             endpoint = getattr(self, 'enterprise-learner')  # pylint: disable=literal-used-as-attribute
             response = endpoint.get(**querystring)
         except (HttpClientError, HttpServerError) as exc:
-            LOGGER.warning("Unable to retrieve Enterprise Customer Learner details for user {}: {}"
-                           .format(user.username, exc))
+            LOGGER.warning((
+                "[Data Overview Failure] Unable to retrieve Enterprise Customer Learner details. User: {user}".format(
```
Comment (brittneyexline) [inline enterprise_data/clients.py]: can we include the exception message as well?

File: enterprise_data/clients.py [hunk 2]
```diff
@@ -79,8 +92,13 @@ def get_enterprise_customer(self, user, enterprise_id):
             endpoint = endpoint(enterprise_id)
             response = endpoint.get()
         except (HttpClientError, HttpServerError) as exc:
-            LOGGER.warning("Unable to retrieve Enterprise Customer details for user {}: {}"
-                           .format(user.username, exc))
+            LOGGER.warning((
```
Comment (brittneyexline) [inline enterprise_data/clients.py:97]: same here, please include the exception message

**Review by brittneyexline — APPROVED** [review]
just a few comments around including the exception, but once that is added you can merge!

**Pull request merged**